tests/pipeline/test_classifier.py

```diff
@@ -607,6 +607,48 @@ class C(Classifier):
             ),
         )
 
+    @parameter_space(
+        dtype_and_missing=[(int64_dtype, -1), (categorical_dtype, None)],
+    )
+    def test_peer_count(self, dtype_and_missing):
+        class C(Classifier):
+            dtype = dtype_and_missing[0]
+            missing_value = dtype_and_missing[1]
+            inputs = ()
+            window_length = 0
+
+        c = C()
+
+        if dtype_and_missing[0] == int64_dtype:
+            data = np.array(
+                [[2, 4, 4, -1, 3, 4, -1, 1, 3]],
+                dtype=int64_dtype,
+            )
+        else:
+            data = LabelArray(
+                [['aa', 'a', 'a', None, 'b', 'a', None, 'c', 'b']],
+                missing_value=None,
+            )
+
+        expected = np.array(
+            [[1, 3, 3, 2, 2, 3, 2, 1, 2]],
+            dtype=int64_dtype
+        )
+
+        terms = {
+            'peer_counts': c.peer_count(),
```

Comment:
I wonder if it's also worth asserting that the result of constructing a `PeerCount` explicitly also matches the expected results? Probably not, but figured I'd throw that idea out there.

Reply:
Good idea, I think I could do that pretty easily

Reply:
Decided to not incorporate that in the test since atm I'm thinking PeerCount should only get accessed by a .peer_count() call.  Could very easily add the test, probably would want to put it in test_factor though.

Reply:
Ok, considering `peer_count` is just a thin wrapper around `PeerCount` anyway, the coverage is already here.